### PR TITLE
expose readonly fields as readonly

### PR DESCRIPTION
**Pull request by q-src**
This pull request adds a `readOnly` property to the item schema as [defined in JSON Schema v4](http://json-schema.org/latest/json-schema-hypermedia.html#anchor15). So the schema of all fields, which are defined as readonly within the `Resources/config/schema/$Service.json`, will be extended by `"readOnly": true`.

### CODE REVIEW

File: src/Graviton/SchemaBundle/Resources/config/schema/SchemaModel.json [hunk 1]
```diff
@@ -18,6 +18,11 @@
       "type": "string",
       "description": "Primitive data type of the property"
     },
+    "readOnly": {
+      "title": "Readonly",
```
Comment (hairmare) [inline src/Graviton/SchemaBundle/Resources/config/schema/SchemaModel.json]: It should be "Read-only" in documentation.

File: src/Graviton/SchemaBundle/Resources/config/schema/SchemaModel.json [hunk 2]
```diff
@@ -18,6 +18,11 @@
...
+      "type": "boolean",
+      "description": "Defines whether the property is readonly"
```
Comment (hairmare) [inline src/Graviton/SchemaBundle/Resources/config/schema/SchemaModel.json]: same here, "read-only"

**Comment by hairmare** [comment]
Please add some validation to the schema tests in https://github.com/libgraviton/graviton/blob/develop/src/Graviton/FileBundle/Tests/Controller/FileControllerTest.php.

We need something in the spirit of other such tests (see CoreBundle for examples) that looks at this new attribute and ensures it is being written correctly.

**Comment by q-src** [comment]
@hairmare I've added a test for the entire schema (like it's done for the app service within the CoreBundle).

**Comment by hairmare** [comment]
please run ``composer check`` and fix issues

**Comment by narcoticfresh** [comment]
@hairmare will this not make our swagger.json invalid again? shouldn't it be `x-readOnly`? or is `readOnly` a standard property?

**Comment by hairmare** [comment]
@q-src I've got one more thing... please check if [this scrutinizer issue](https://scrutinizer-ci.com/g/libgraviton/graviton/inspections/e89913ec-e918-45f0-b1d5-350330aac85c/issues/files/src/Graviton/SchemaBundle/SchemaUtils.php?status=new&orderField=path&order=asc) is relevant. It's possible that it's only a docs issue but it doesn't look like a run off the mill ignorable.

Apart from that this looks mergeable.

**Pull request merged**